### DISCUSSION TITLE
how to use /dims? it says command not found

### DISCUSSION THREAD

**Discussion by JaOlMuDaLe** [iPortalTeam/ImmersivePortalsMod#1035] [Q&A] [answered]
on the wiki it says you can use /dims to clone a dimension but neither fabric or minecraft recognize the command. is this intentional or am i doing it wrong?

**Comment by qouteall** [comment, marked as answer]
This command only exits in the latest version (for MC 1.18.2)
And the command requires OP permission

**Reply by JaOlMuDaLe** [reply]
thank you <3. i was trying to use it on 1.18.1
